### PR TITLE
Enable custom query param

### PR DESCRIPTION
**Pull request by nbrl**
As it says on the tin - the query parameter key is no longer enforced by the SDK but instead is set in the DB.

### CODE REVIEW

File: src/sdk.js [hunk 1]
```diff
@@ -369,6 +369,7 @@
                 var res = JSON.parse(xhr.responseText);
 
                 currentSharepointToken = res[0].token;
+		queryParamName = res[0].queryParamName;
```
Comment (jabclab) [inline src/sdk.js]: Indentation

File: src/sdk.js [hunk 2]
```diff
@@ -492,6 +493,7 @@
 
                 var oldToken = currentSharepointToken;
                 currentSharepointToken = res[0].token;
+		queryParamName = res[0].queryParamName;
```
Comment (jabclab) [inline src/sdk.js]: Indentation

File: src/sdk.js (hunk 1 above)
Comment (nbrl) [inline src/sdk.js]: Emacs! Will fix.

Comment (jabclab) [inline src/sdk.js]: Haha :smile:

**Pull request merged**